ditto/base.json

```diff
@@ -511,6 +511,14 @@
   "text_626c0c301a16a600ea061479": "Email",
   "text_626c0c301a16a600ea06147d": "Phone number",
   "text_626c0c301a16a600ea06148d": "Shipping information",
+  "text_6335e50b0b089e1d8ed50960": "{{plan_name}} is planned be active on the Jul. 13, 2023",
```

Comment:
Feel that this translation is wrong. Should be `planName` and probably Jul. 13, 2023 should not be hardcoded 🤔

Reply:
Yeah seem you use `planName` and `startDate` on invocation